bin/fmfexporter

```diff
@@ -54,4 +56,12 @@ if __name__ == '__main__':
         sys.exit(0)
 
     # Submitting filtered test cases
-    adapter.submit_testcase_list(tc_list)
+    submitted_tc_list = adapter.submit_testcases(tc_list, parsed.jira_populate_tc)
+
+    # Linking Test Case Work items in jira
+    if parsed.jira_config is not None:
+        jira_pop = FMFJiraPopulator(parsed.jira_config)
+        jira_pop.populate_testcases(submitted_tc_list)
+    else:
+        LOGGER.warning("Jira configuration not provided")
+
```

Comment:
The FMFAdapter interface does not expect submit_testcases method to return any value (see below). The lines above be moved to `fmf_adapter_polarion.py` under submit_test_cases method (as jira connector has been made internal to it).

```
    @abc.abstractmethod
    def submit_testcases(self, fmf_testcases: list, parse_response=False):
```